targets/targets.json

```diff
@@ -4577,6 +4580,7 @@
             "PORTINOUT",
             "PORTOUT",
             "PWMOUT",
+	    "SD",
```

Comment:
please fix the alignment here

Reply:
@0xc0170 
Opened up targets.json in github's file editor. It seemed it was aligned properly to all the other lines.
Copied that line from Mbed Studio, that might have handled indentation different from what github expects.
Now i deleted any tabbing for that line (at some point it jumped several characters at once, so there might have been a special tabbing) and indented that line with plain space characters. Hope it is fine now.